google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/standard/AppEngineStandardDeploymentEditor.java

```diff
@@ -236,27 +143,27 @@ protected JComponent createEditor() {
   }
 
   @VisibleForTesting
-  JCheckBox getPromoteCheckbox() {
-    return promoteCheckbox;
+  JCheckBox getStopPreviousVersionCheckbox() {
+    return commonConfig.getStopPreviousVersionCheckbox();
   }
 
   @VisibleForTesting
-  JCheckBox getStopPreviousVersionCheckbox() {
-    return stopPreviousVersionCheckbox;
+  public JCheckBox getDeployAllConfigsCheckbox() {
```

Comment:
Do these need to be public? I don't see where they're used in a test outside this package.